dds_web/database/models.py

```diff
@@ -100,6 +100,39 @@ class ProjectInviteKeys(db.Model):
     key = db.Column(db.LargeBinary(300), nullable=False, unique=True)
 
 
+class ProjectInvites(db.Model):
```

Comment:
I think we could do it without this table, and instead just add "owner" to the ProjectInviteKeys instead or is there some issue with primary keys there?

Reply:
I can see how I caused this confusion. Sorry for that. I think it should be fairly quick to delete it though?

Reply:
Yeah I agree, I think it's better to add this to the `ProjectInviteKeys` table.

Reply:
Otherwise we have `Invite`, `ProjectInviteKeys` and `ProjectInvites`

Reply:
It does mean that we would have the `owner` column for all unit users too though, but since we wouldn't need to check that in that case it doesn't actually matter

Reply:
Sorry, but this is a clear **NO** from my side!. It was _my_ suggestion last week to have the owner column on the keys instead of the Project/User association and this was repeatedly turned down by you. I even stressed that turning my proposal down would require this additional association table.  

I yesterday went almost mad solving all the little tiny issues arising from this way of building it. It caused me ~6h of despair to address the "Project Owner problem" and how to resolve the joins. 

Having the owner column on the keys for the invites but on the ProjectUser association table for the Users would imply changing a ton of function that need to process instances of both `Invite` and `User` objects. If we are consistent in this regard, we can be basically use the same code.

Reply:
I see that I didn't make myself clear regarding this before. I agreed that we shouldn't push for the owner column solution on the previous PR on the invite keys, but instead do it in a new PR. Again, sorry for not being clear on this point. However, if you now think that the the ProjectInvites table is a better solution, I'm all ears.

Reply:
No, I am not at all saying that the `ProjectInvites` table is the better solution (I still like my initial proposal more), but it is the solution that we have decided on and that I thus implemented. 

The only way to make this whole thing even more complicated is solving the same problem in two different ways for `Invites` and `Users`.